Sources/Sentry/SentryEnvelopeItemHeader.m

```diff
@@ -11,6 +11,16 @@ - (instancetype)initWithType:(NSString *)type length:(NSUInteger)length
     return self;
 }
 
+- (instancetype)initWithType:(NSString *)type
```

Comment:
l: The initializer with  filename could use this initializer instead of the other one.